modules/sbt-dependency-manager/src/test/scala-2.11/notebook/util/CoursierDepsTest.scala

```diff
@@ -0,0 +1,53 @@
+package notebook.util
+
+import com.datafellas.utils.Repos
+import coursier.{Attributes, Dependency, Module}
+import coursier.maven.MavenRepository
+import org.scalatest._
+import org.sonatype.aether.repository.RemoteRepository
+
+// FIXME: missing scala-style exclusions
+class CoursierDepsTest extends WordSpec with Matchers with BeforeAndAfterAll with Inside {
+  val sparkVersion = "10.0.0"
+  val version = "0.1.1"
+  val classifier = "some-classifier"
+  val scalaVer = "2.11"
+
+  // based on ReplCalculator
+  val remotes: List[RemoteRepository] = List(Repos.mavenLocal, Repos.central, Repos.sparkPackages, Repos.oss)
+
+  val (repos, deps) = CoursierDeps.parseCoursierDependencies(
+    cp =
+      s"""|com.someorg % java-artifact % $version
+          |com.someorg %% scala-artifact % $version
+          |com.someorg %% scala-artifact-cl % $version classifier $classifier
+          |- org.excludeme : excluded-artifact-mavenlike : _
+          |- org.excluded.org : _ : _
+          |- org.excludeme % excluded-artifact-scalalike % _
```

Comment:
this is type of exclusion is not supported right now (see `expectedExcludes`). bug in existing `scala 2.11/Deps.scala`